src/locales/cs.ts

```diff
@@ -204,141 +204,140 @@ export default {
-  //   our_lord_of_miracles: 'Our Lord of Miracles',
+    our_lord_of_miracles: 'Pána zázraků',
     presentation_of_mary: 'Zasvěcení Panny Marie v Jeruzalémě',
     queenship_of_mary: 'Panny Marie Královny',
     adalbert_of_prague_bishop: 'Sv. Vojtěcha, biskupa a mučedníka',
-  //   aelred_of_rievaulx_abbot: 'Saint Aelred of Rievaulx, Abbot',
-  //   aengus_of_tallaght_bishop: 'Saint Aengus, Bishop and Abbot',
+    aelred_of_rievaulx_abbot: 'Sv. Aelreda z Rievaulx, opata',
+    aengus_of_tallaght_bishop: 'Sv. Aengusa, biskupa a opata',
     agatha_of_sicily_virgin: 'Sv. Agáty, panny a mučednice',
     agnes_of_bohemia_virgin: 'Sv. Anežky České, panny',
-  //   agnes_cao_guiying_martyr: 'Saint Agnes Cao Guiying, Martyr',
+    agnes_cao_guiying_martyr: 'Sv. Anežky Cao Guiying, mučednice',
     agnes_of_rome_virgin: 'Sv. Anežky Římské, panny a mučednice',
-  //   aidan_of_ferns_bishop: 'Saint Aidan, Bishop',
-  //   aidan_of_lindisfarne_bishop_and_the_saints_of_lindisfarne: 'Saint Aidan, Bishop, and the Saints of Lindisfarne',
-  //   aidan_of_lindisfarne_bishop: 'Saint Aidan of Lindisfarne, Bishop and Missionary',
-  //   ailbe_of_emly_bishop: 'Saint Ailbe, Bishop',
-  //   alban_of_britain_martyr: 'Saint Alban, Martyr',
-  //   alberic_crescitelli_priest: 'Saint Alberic Crescitelli, Priest and Martyr',
-  //   albert_chmielowski_religious: 'Saint Albert Chmielowski, Religious',
-  //   alberto_hurtado_priest: 'Saint Alberto Hurtado, Priest',
+    aidan_of_ferns_bishop: 'Sv. Aidana, biskupa',
+    aidan_of_lindisfarne_bishop_and_the_saints_of_lindisfarne: 'Sv. Aidana, biskupa, a lindisfarnských svatých',
+    aidan_of_lindisfarne_bishop: 'Sv. Aidana Lindisfarnského, biskupa a misionáře',
+    ailbe_of_emly_bishop: 'Sv. Ailbe, biskupa',
+    alban_of_britain_martyr: 'Sv. Albana, mučedníka',
+    alberic_crescitelli_priest: 'Sv. Alberika Crescitelliho, kněze a mučedníka',
+    albert_chmielowski_religious: 'Sv. Alberta Chmielowského, řeholníka',
+    alberto_hurtado_priest: 'Sv. Alberta Hurtada, kněze',
     albert_the_great_bishop: 'Sv. Alberta Velikého, biskupa a učitele církve',
     aloysius_gonzaga_religious: 'Sv. Aloise Gonzagy, řeholníka',
-  //   alphonsa_of_the_immaculate_conception_muttathupadathu_virgin: 'Saint Alphonsa of the Immaculate Conception Muttathupadathu, Virgin',
+    alphonsa_of_the_immaculate_conception_muttathupadathu_virgin: 'Sv. Alfonsy od Neposkvrněného početí Muttathupadathu, panny',
     alphonsus_liguori_bishop: 'Sv. Alfonsa Marie z Liguori, biskupa a učitele církve',
-  //   amand_of_maastricht_bishop: 'Saint Amand, Bishop',
+    amand_of_maastricht_bishop: 'Sv. Milana, biskupa',
     ambrose_of_milan_bishop: 'Sv. Ambrože, biskupa a učitele církve',
-  //   andre_bessette_religious: 'Saint André Bessette, Religious',
-  //   andrew_bobola_priest: 'Saint Andrew Bobola, Priest and Martyr',
+    andre_bessette_religious: 'Svatého Andreje Alfréda Bessette, řeholníka',
```

Comment:
I'm not sure if the French name `André` would be bohemized, but if so, the correct Czech version is `Ondřeje`